docs/guides/importing-private-repositories.rst

```diff
@@ -0,0 +1,125 @@
+Manually Importing Private Repositories
+=======================================
+
+.. warning::
+
+   This guide is for :doc:`/commercial/index`.
+
+If you are using one of our supported integrations (GitHub, GitLab, or Bitbucket),
+we recommend :doc:`connecting your account </connected-accounts>` and importing the project from
+https://readthedocs.com/dashboard/import.
+If you are using an unsupported integration, or don't want to connect your account,
+you'll need to do some extra steps in order to have your project working.
+
+#. Manually import your project using an SSH URL
+#. Allow access to your project using an SSH key
+#. Setup a webhook to build your documentation on every commit
```

Comment:
with a definition list, items are over-spaced, well, tried putting the number as the definition and the text as content. I have changed the items to be marked as bold instead, if you have other ideas how to use the def list let me know.

Reply:
I can't find a good example of what I'm describing. The bold definitely helps, and I'm not sure if a definition list is necessarily the answer either.

What I've done is something in between a local TOC and definition list for this sort of scenario -- that is, lay out the steps and describe them just a little bit. The additional description on each step provides just a little more hand holding for the user, and it replaces the local TOC.

If you wanted to play around with it more, here is what I'm describing:

```rst
To accomplish the thing you need to do, there are a few steps that you will need
to take:

`The first step you'll take`_
    This is a very important step due because a, b, and c

`The next step`_
    You can't miss this step. This step is important because your project
    won't do the thing you need to do without it

`The final step`_
    Lastly, you definitely can't forget this step, because of other reasons

The first step you'll take
--------------------------

The next step
-------------

The final step
--------------
```